build/linux/installer/scripts/tomlparser-health-config.rb

```diff
@@ -13,32 +13,83 @@
 @configMapMountPath = "/etc/config/settings/agent-settings"
 @configSchemaVersion = ""
 @enable_health_model = false
+@nodesChunkSize = 0
```

Comment:
Cleaner to define the default here ( instead of zero ) , and always set it (to default if not config map), and remove the defaults in each plugin, so this is all in one place.

Reply:
Done.